src/Makefile_obj.in

```diff
@@ -91,7 +91,7 @@ endif
 #CCMALLOC = /usr/local/lib/ccmalloc-gcc.o -lccmalloc -ldl
 
 # -lfl not needed as Flex invoked with %nowrap option
-LIBS = $(CFG_LIBS) -lm
+LIBS = $(CFG_LIBS) -lm -lrt
```

Comment:
Normally libraries would need a test in configure, but librt should be standard so think we are ok.

Reply:
Good you mention this. librt is standardized in POSIX, in the same part of the standard as `aio_write`. But as you mention below, `aio_write` is not available in MSVC. I'll add a **configure check for aio_write + librt**. Without `aio_*` it is not needed anyway.